File: CodeNite-1/Solutions/Python/A_Angry_Professor_at_USICT.py
#author: Tushar Khanduri
n = int(input())
a = input()
a = a.split(" ")
zero,one = 0,0

for i in a:
    if i=='0':
        zero+=1
    else:
        one+=1
if one>=zero:
    print("Yes")
else:
    print("No")

File: CodeNite-1/Solutions/C++/A_Angry_Professor_at_USICT.cpp
#include <stdio.h>

int main()
{
    int n;
    scanf("%d", &n);

    int c0 = 0;
    int c1 = 0;
    for (int i = 0; i < n; i++)
    {
        int x;
        scanf("%d", &x);

        if (x == 0)
            c0++;
        else
            c1++;
    }

    if (c0 <= c1)
        printf("Yes\n");
    else
        printf("No\n");
}